pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -34,11 +36,14 @@ import (
 
 var bridgeFakeIP = "169.254.75.86/32"
 
+// DefaultProtocol is the default port protocol
+const DefaultProtocol string = "TCP"
+
+// DefaultVMCIDR is the default CIRD for vm network
+const DefaultVMCIDR = "10.0.2.0/24"
```

Comment:
Seems like it's slirp specific. If so, shouldn't the name reflect that?

Reply:
It's not slirp specific, we can use it on any interface solution the need to create a network subnet inside the pod.
like proxy solution or nat.

Reply:
Ack. BTW what's the rationale to allow users change the CIDR? Do we envision some images not working with our CIDR for some reason?

Reply:
Because we may conflict with there network subnets.